src/sql/store.py

```diff
@@ -81,6 +81,12 @@ def __init__(self, store: SQLStore, query: str, with_: Iterable = None):
         self._query = query
         self._with_ = with_ or []
 
+        if any(map(lambda x: "-" in x, self._with_)):
+            raise FutureWarning(
```

Comment:
warnings are displayed using warnings.warn: https://docs.python.org/3/library/warnings.html - here you're raising an error

Reply:
Just made this modification.